internal/nhooyr.io/websocket/conn.go

```diff
@@ -185,6 +193,9 @@ func (c *Conn) close(err error) {
 }
 
 func (c *Conn) timeoutLoop() {
```

Comment:
`activityProbe` is used to monitor reads and writes activity until the connection is closed.